pyomo/contrib/viewer/qt.py

```diff
@@ -38,17 +38,19 @@
 import_errors = []
 # Set this to the Qt wrapper module is available
 available = False
-
-for module_str in supported:
-    try:
-        qt_package = importlib.import_module(module_str)
-        QtWidgets = importlib.import_module(f"{module_str}.QtWidgets")
-        QtCore = importlib.import_module(f"{module_str}.QtCore")
-        QtGui = importlib.import_module(f"{module_str}.QtGui")
-        available = module_str
-        break
-    except Exception as e:
-        import_errors.append(f"{e}")
+from pyomo.core.base.units_container import pint_available
+
+if pint_available:
+    for module_str in supported:
+        try:
+            qt_package = importlib.import_module(module_str)
+            QtWidgets = importlib.import_module(f"{module_str}.QtWidgets")
+            QtCore = importlib.import_module(f"{module_str}.QtCore")
+            QtGui = importlib.import_module(f"{module_str}.QtGui")
+            available = module_str
+            break
+        except Exception as e:
+            import_errors.append(f"{e}")
```

Comment:
Why are we marking "qt" as not available due to pint?  I would rather see us leave this as-is, and just skip the tests that rely on pint if pint is not available.

Reply:
Changed